Samples/Chirper/ChirperGrains/ChirperAccount.cs

```diff
@@ -31,7 +31,6 @@ public class ChirperAccountState
         public string UserAlias { get; set; }
     }
 
-    [Reentrant]
```

Comment:
You get etag update violations since the grain is reentant and as a result you may be doing parallel calls to write state, which race with each other in the storage. That is a general issue with state persistence and reentrant grains.
Luckily, the fix is easy.
There is an (easy) way to write the code such that the grain is still reentrant, but all writes to storage are serialized. I have written it once. I will find it out and share here.